docker/nginx.conf

```diff
@@ -2,44 +2,41 @@
 # metering turned on and demonstrates the use of some of the configuration options for the metering module.
 
 worker_processes  auto;
-worker_rlimit_nofile 50000;
-
 # error_log   /tmp/error.log debug;
 
-load_module modules/metered_access_module.so;
-
 events {
-  worker_connections  32;
+  worker_connections 32;
   use epoll;
 }
 
 http {
-  include       mime.types;
   default_type  application/octet-stream;
-  types_hash_max_size 4096;
-  server_names_hash_bucket_size 128;
-
   sendfile        on;
   keepalive_timeout  65;
-  # send headers in one piece, it is better than sending them one by one
   tcp_nopush on;
-
-  # don't buffer data sent, good for small data bursts in real time
   tcp_nodelay on;
 
   server {
     server_name  localhost;
     access_log off;
     listen 9090;
-    location /nginx_status {
+    location /status {
       stub_status on;
-      metered_access_ipc_fd_path "/tmp/metering-agent-ipc.sock";
-      metered_access	on;
-      metered_access_io_timeout 10;
-      metered_access_extract_header "apiKey";
-      metered_access_extract_header "X-Forwarded-For";
-      metered_access_extract_header "Env";
-      metered_access_servername "demoserver";
+      auth_request /auth;
     }
+
+    location = /auth {
+      proxy_pass "http://metering-agent:3050/authenticate";
```

Comment:
does it mean it will only work with the docker compose and host named `metering-agent`? can we make it configurable?